runners.go

```diff
@@ -77,6 +77,7 @@ type RunnerDetails struct {
 		Name   string `json:"name"`
 		WebURL string `json:"web_url"`
 	} `json:"groups"`
+	Locked         bool `json:"locked"`
```

Comment:
These fields are (more or less) ordered according to the docs, so in that case this field should be above `AccessLevel`. And could you also add the field to the create and update option structs to make support for the field complete? Thanks!

Reply:
I've moved it up; AFAICT the update and create structs already have `Locked`, though I might be missing something.

Reply:
Your absolutely right! Thanks for fixing the PR!